docs/resources/dcs_redis_instance.md

```diff
@@ -0,0 +1,261 @@
+  Redis 5.0 instances and not by Redis 3.0 instance.
+  The valid commands that can be renamed are: *command*, *keys*, *flushdb*, *flushall* and *hgetall*.
+
+* `tags` - (Optional, Map) The key/value pairs to associate with the redis instance.
+
+* `charging_mode` - (Optional, String, ForceNew) Specifies the charging mode of the redis instance.
+  The valid values are as follows:
+  + `prePaid`: indicates the yearly/monthly billing mode.
+  + `postPaid`: indicates the pay-per-use billing mode.
+  Changing this creates a new instance.
+
+* `period_unit` - (Optional, String, ForceNew) Specifies the charging period unit of the instance.
+  Valid values are *month* and *year*. This parameter is mandatory if `charging_mode` is set to *prePaid*.
+  Changing this creates a new instance.
+
+* `period` - (Optional, Int, ForceNew) Specifies the charging period of the instance.
+  If `period_unit` is set to *month*, the value ranges from 1 to 9.
+  If `period_unit` is set to *year*, the value ranges from 1 to 3.
+  This parameter is mandatory if `charging_mode` is set to *prePaid*.
+  Changing this creates a new instance.
+
+* `auto_renew` - (Optional, String, ForceNew) Specifies whether auto renew is enabled.
+  Valid values are `true` and `false`, defaults to `false`.
+  Changing this creates a new instance.
+
+The `whitelists` block supports:
+
+* `group_name` - (Required, String) Specifies the name of IP address group.
+
+* `ip_address` - (Required, List) Specifies the list of IP address or CIDR which can be whitelisted for an instance.
+  The maximum of ip address is 20.
+
+The `backup_policy` block supports:
+
+* `backup_type` - (Optional, String) Backup type. Default value is `auto`. The valid values are as follows:
+  + `auto`: automatic backup.
+  + `manual`: manual backup.
+
+* `save_days` - (Required, Int) Retention time. Unit: day, the value ranges from 1 to 7.
+
+* `period_type` - (Optional, String) Interval at which backup is performed. Default value is `weekly`.
+  Currently, only weekly backup is supported.
+
+* `backup_at` - (Required, List) Day in a week on which backup starts, the value ranges from 1 to 7.
+  Where: 1 indicates Monday; 7 indicates Sunday.
+
+* `begin_at` - (Required, String) Time at which backup starts.
+  Format: `hh24:00-hh24:00`, "00:00-01:00" indicates that backup starts at 00:00:00.
+
+* `timezone_offset` - (Optional, String) Time zone in which backup is performed.
+  The value ranges from GMT–12:00 to GMT+12:00.
+  If omitted, the current time zone of the DCS-Server VM is used by default.
+
+## Attributes Reference
+
+In addition to all arguments above, the following attributes are exported:
+
+* `id` - A resource ID in UUID format.
+
+* `status` - Cache instance status. The valid values are as follows:
+  + `CREATING`: Creating is the status before the Running state.
+  + `CREATEFAILED`: The DCS instance failed to be created.
```

Comment:
CREATING and CREATEFAILED should not be here

Reply:
done